common/tools/dev-tool/launch.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const path = require("path");
+
+if (process.env.DEBUG) {
+  console.info("Azure SDK for JS dev-tool: bootstrapping from", __dirname);
+}
+
+// Shim to invoke true typescript source
+require("ts-node").register({
```

Comment:
Would it be possible to compile things instead of using ts-node?

Reply:
I believe this was done to avoid needing to build the devtool itself before being able to run any other task in the tree.

Reply:
gotcha